### ISSUE TITLE
Make color-contrast failures on role tab elements with tabindex=-1 and opacity property as needs review

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#2447]
Currently, role ```tab``` elements with tabindex=-1 and with opacity at a certain level fail ```color contrast``` rule. But in cases like 
 breadcrumb menus, opacity might be used to depict parts of the menu as inactive. Please consider making such failure instances as needs review instead of failures

Expectation: Do not fail this use case for ```color-contrast``` rule and show it as needs review

Actual: This instance fails ```color-contrast``` rule

Motivation: Opacity might be used to style a few controls in a certain way to depict them as inactive controls

Demo link: https://codepen.io/iamrafan/pen/abNdLyN

<pre><code>axe-core version: 3.5.2-canary.37be85f
axe-Coconut version: 4.4.1</code></pre>

**Comment by WilcoFiers** [comment]
I don't think I agree with you on that one. Technically, this isn't a color contrast issue, but this button doesn't properly express its state, which is still a failure of WCAG (4.1.2 specifically). Putting `tabindex="-1"` on a button doesn't disable it. Screen reader users will still have access to it. You need to give that button `aria-disabled="true"` to communicate the intent. If you do that, axe-core will ignore the button aswel.

**Comment by iamrafan** [comment]
Sure, I agree that ```aria-disabled``` must be used, but I filed this issue because it looks like a false positive to fail 'color-contrast' on this use case.


I am closing this issue. Will re-open it later, if requried.
